adafruit_minimqtt/adafruit_minimqtt.py

```diff
@@ -668,11 +692,11 @@ def loop_forever(self):
         method if you want to run a program forever.
         Code below a call to this method will NOT execute.
 
-        NOTE: This method is depreciated and will be removed in the
-        next major release. Please see examples/minimqtt_pub_sub_blocking.py
-        for an example of creating a blocking loop which can handle wireless
-        network events.
-
+        .. note:: This method is depreciated and will be removed in the
```

Comment:
> I noticed there was a (un-admonished) deprecation notice and a reference to an example that was not being displayed on RTD.

In the next PR (with the `fixed_header` change), could you also remove this method?

Reply:
Yes; can do. Prepping for the next "major" release? No planned "minor" releases in immediate future? Is the [Bluetooth PR](#15 ) such a significant change? I also see you have a branch for compatibility with CPython sockets...

Reply:
Opened #45 to satisfy request

Reply:
@2bnd5 re: I'm not ready to merge the CPython sockets branch just yet, but yes, that'll be a major release.